packages/playback-controls/index.d.ts

```diff
@@ -0,0 +1,4 @@
+import PlaybackControls from './lib/playback-controls.d';
+import { PlaybackMode } from './lib/playback-mode.d';
```

Comment:
question: what does the `d` stand for?

Reply:
TypeScript generates [Declaration Files](https://www.typescriptlang.org/docs/handbook/declaration-files/introduction.html) that allow other TypeScript projects to understand what types of objects it is dealing with. Since TypeScript projects get compiled to JavaScript when they are installed, they lose their type information. The `.d.ts` file keeps that type information around so the TS consumers can know what they're dealing with.